### DISCUSSION TITLE
Problem downloading the package as a dependency to the project

### DISCUSSION THREAD

**Discussion by Dominic263** [pointfreeco/swift-composable-architecture#1146] [Q&A]
Has anyone else faced a problem, adding the TCA package to their XCode project? When I click on add package nothing happens; basically the fetch seems to be in an endless loop.

**Comment by stephencelis** [comment]
@Dominic263 Have you tried resetting package caches or updating packages to give SwiftPM a little kick?

<img width="388" alt="image" src="https://user-images.githubusercontent.com/658/173879672-7e446b9a-dc0b-4eca-a7f0-fed248f5b147.png">

You might be experiencing an Xcode bug. There's a SwiftPM Slack where some Apple engineers that work on SPM and Xcode hang out. You could maybe get them to help troubleshoot there:

https://swift-package-manager.herokuapp.com
